lib/recipients/establishment.js

```diff
@@ -2,79 +2,137 @@ const { get } = require('lodash');
+  const notifyPelh = params => {
+    logger.verbose(`${params.logMsg}, notifying establishment licence holder(s)`);
+    pelh.forEach(lh => {
+      if (!notifications.has(lh.id)) {
+        notifications.set(lh.id, { ...params, recipient: lh });
       }
+    });
+  };
 
-      if (task.status === 'returned-to-applicant' || taskHelper.isOverTheFence(task) || taskHelper.isClosed(task)) {
-        return getAdmins(establishmentId)
-          .then(admins => admins.filter(subscribedFilter({ establishmentId, licenceType: 'pel' })))
-          .then(admins => {
-            admins.map(profile => {
-              if (!notifications.has(profile.id)) {
-                if (task.status === 'returned-to-applicant') {
-                  logger.verbose('task is with admin, notifying admin');
-                  notifications.set(profile.id, { emailTemplate: 'task-action-required', applicant, recipient: profile });
-
-                } else if (taskHelper.isOverTheFence(task)) {
-                  logger.verbose('task is over the fence, notifying admin');
-                  notifications.set(profile.id, { emailTemplate: 'task-with-asru', applicant, recipient: profile });
-
-                } else if (taskHelper.isGranted(task)) {
-                  logger.verbose('licence granted, notifying admin');
-                  const template = ['place', 'role'].includes(model) ? 'licence-amended' : 'licence-granted';
-                  notifications.set(profile.id, { emailTemplate: template, applicant, recipient: profile });
-
-                } else if (taskHelper.isClosed(task)) {
-                  logger.verbose('task is closed, notifying admin');
-                  notifications.set(profile.id, { emailTemplate: 'task-closed', applicant, recipient: profile });
-                }
-              }
-            });
-          });
+  const notifyAdmins = params => {
+    logger.verbose(`${params.logMsg}, notifying establishment admins`);
+    admins.forEach(admin => {
+      if (!notifications.has(admin.id)) {
+        notifications.set(admin.id, { ...params, recipient: admin });
       }
-    })
-    .then(() => notifications);
+    });
+  };
+
+  const params = {
+    establishmentId,
+    licenceNumber: establishment.licenceNumber,
+    applicant
+  };
+
+  if (applicant) {
+    logger.debug(`applicant is ${applicant.firstName} ${applicant.lastName}`);
+  }
+
+  if (taskHelper.isReminderNotice(task)) {
+    const when = get(task, 'data.when');
+
+    const reminderParams = {
+      ...params,
+      modelType: 'establishment',
+      emailTemplate: 'condition-reminder',
+      logMsg: `establishment condition is due ${when}`,
+      addTaskTypeToSubject: false,
+      when,
+      deadline: task.data.reminder.deadline,
+      identifier: `${establishmentId}-${action}`
+    };
+
+    notifyPelh(reminderParams);
+    notifyAdmins(reminderParams);
+
+    return notifications;
```

Comment:
this is the new bit - the rest is a refactor to remove the nesting and make it easier to follow.